packages/dev/core/src/Debug/skeletonViewer.ts

```diff
@@ -556,7 +566,15 @@ export class SkeletonViewer {
         let boneNum = 0;
 
         const mesh = this.mesh;
-        const meshPos = mesh.position;
+        let transformNode;
+        let meshPos;
+        if (mesh) {
+            transformNode = mesh;
+            meshPos = mesh.position;
+        } else {
+            transformNode = new TransformNode("");

```

Comment:
You should remove this transform node when not used anymore, to avoid leaks.

Reply:
@Popov72 I am not sure if I get it. Do you mean `transformNode = null` after the last operation?

Reply:
No, `transformNode.dispose()`, to remove the node from the scene graph and release all ressources.

Reply:
@Popov72 Thanks for clarifying, done!